### ISSUE TITLE
Feature request - Add BBCode to default formats

### ISSUE THREAD

**Issue by nomind60s** [ku/CreateLink#33]
Please consider adding [BBCode](https://en.wikipedia.org/wiki/BBCode) to the default formats. Something like:

**Name:** BBCode
**Format:** `[url=%url%]%htmlEscapedText%[/url]`

**Comment by CommSysEng** [comment]
I agree with this request. Please add [BBCode](https://en.wikipedia.org/wiki/BBCode) to the default format
